ui/core/components/pickers/list_picker.tsx

```diff
@@ -246,10 +282,14 @@ export class ListPicker<ModObject, ItemType> extends Input<ModObject, Array<Item
 			itemContainer.addEventListener(
 				'dragenter',
 				event => {
-					if (!curDragData || curDragData.listPicker != this) {
+					if (
+						!curDragData ||
+						curDragData.listPicker.config.itemLabel !== this.config.itemLabel ||
```

Comment:
Can this specific if check become a helper function? I see it repeated a lot (not that the old one wasn't)

Reply:
Yea I could do that 👍